tests/core/test_streams.py

```diff
@@ -48,6 +48,16 @@ def get_records(self, context: Optional[dict]) -> Iterable[Dict[str, Any]]:
         yield {"id": 3, "value": "India"}
 
 
+class UnixTimestampIncrementalStream(SimpleTestStream):
+    name = "unix_ts"
+    schema = PropertiesList(
+        Property("id", IntegerType, required=True),
+        Property("value", StringType, required=True),
+        Property("updatedAt", IntegerType, required=True),
+    ).to_dict()
+    replication_key = "updatedAt"
```

Comment:
As of now, I believe we only accept ISO timestamp strings and not Unix timestamps (integers). (Someone please correct me if that's not correct.)

Would it be okay to keep Unix timestamps out of scope for this and to discuss those in another issue/PR?

Reply:
@aaronsteers ditto my other comment